bcipy/task/paradigm/rsvp/calibration/calibration.py

```diff
@@ -93,6 +95,36 @@ def generate_stimuli(self):
                                              is_txt=self.rsvp.is_txt_stim,
                                              color=self.color)
 
+    def convert_timing_triggers(self, timing: List[tuple]) -> List[Trigger]:
+        """Convert Stimuli Times to Triggers.
+
+        Using the stimuli presentation times provided by the display, convert them into BciPy Triggers.
+        """
+        triggers = []
+        x = 0
+
+        # loop over timing and assign the correct TriggerType
+        for letter, time in timing:
+            if x == 0:
+                targetness = TriggerType.PROMPT
+                target_letter = letter
+            elif x == 1:
+                targetness = TriggerType.FIXATION
+            elif x > 1 and target_letter == letter:
```

Comment:
What do you think about initializing the `target_letter` outside of the loop?
    `target_letter = timing[0][1] if timing else None`

I found it a bit confusing that it's initialized within a branch and used in the `else` condition, but that's probably my bias showing up from other languages where scoping rules would prevent this.

Reply:
I'm not opposed to doing this. I just copied the same logic we used before to avoid any unexpected changes.